.gitmodules

```diff
@@ -18,20 +18,20 @@
 [submodule "wrangler-transform"]
 	path = wrangler-transform
 	url = ../../data-integrations/wrangler.git
-	branch = develop
+	branch = release/4.2
```

Comment:
intentional?

Reply:
no, I did a blind merge. Thanks for catching, will fix